gramineproject/gramine
# [Pal/Linux-SGX] Generate `gsgx.h` using Meson

## Description of the changes 

This PR is on top of #107.

Compared to @woju's original approach, I don't introduce a separate `include/gramine-config.h`, but generate `gsgx.h` directly. This might be less future-proof if we end needing a global config header in the future, but for now it's probably enough?

## How to test this PR? 

Jenkins tests the out-of-tree and DCAP 1.6 drivers, it would be nice to verify compilation on in-tree and DCAP 1.10.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/112)
